static/js/zamboni/collections.js

```diff
@@ -660,8 +660,8 @@ $(document).ready(function () {
                     $widget.setWidth(410);
                     $widget.setPos(ct);
                     $("#id_name").focus();
-                    $widget.delegate('#collections-new-cancel', 'click', loadList)
-                           .delegate('#add-to-collection form', 'submit', handleSubmit);
+                    $('#collections-new-cancel').on('click', loadList);
```

Comment:
You could do event delegation with `on()` if you wanted to and retain the chaining. `on()` optionally takes a selector.